Source/cmake/OptionsGTK.cmake

```diff
@@ -263,6 +262,14 @@ SET_AND_EXPOSE_TO_BUILD(USE_ATSPI 1)
 SET_AND_EXPOSE_TO_BUILD(HAVE_GTK_UNIX_PRINTING ${GTK_UNIX_PRINT_FOUND})
 SET_AND_EXPOSE_TO_BUILD(HAVE_OS_DARK_MODE_SUPPORT 1)
 
+if (USE_SKIA)
+    SET_AND_EXPOSE_TO_BUILD(USE_CAIRO FALSE)
+else ()
+    find_package(Fontconfig 2.13.0 REQUIRED)
+    find_package(Freetype 2.9.0 REQUIRED)
```

Comment:
I wouldn't move these here but keep them at the start. I'm 99% sure I'll be removing the `find_package` calls in Skia's CMake because those libraries aren't used on Windows.

Reply:
We could leave the find package calls there and call them depending on the target platform